demos/demos_by_use_case/bio/ChemicalMappingsDemo.ipynb

```diff
@@ -0,0 +1,886 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "dzmJAwfiAi6k"
+      },
+      "source": [
+        "# Accelerated Chemical Mapping with [Graphistry](graphistry.com)\n",
+        "\n",
+        "This notebook visualizes a chemical dataset describing Blood Brain Barrier Permeability (BBBP) from [MoleculeNet](http://moleculenet.org/datasets-1) and [ECFPs](https://pubs.acs.org/doi/10.1021/ci100050t).\n",
+        "\n",
+        "Using these string formulations of molecular 3D structure we can take advantage of string-based computational algorithms. These string representations look like the following:\n",
+        "\n",
+        "\n",
+        "*   OCC#Cc1cc(Cl)c(C(=O)Nc2ccnc(NC(=O)C3CC3)c2)c(Cl)c1\n",
+        "151276       \n",
+        "*   CCNc1ncnc2c1nc(NC3CCCC3)n2[C@@H]4O[C@H](CO)[C@@H](O)[C@H]4O\n",
+        "172750    \n",
+        "*   CCC(C1=C(O)C2=C(CCCCCC2)OC1=O)c3cccc(NS(=O)(=O)c4ccc(Cl)cc4)c3\n",
+        "155015    \n",
+        "*   CC1CCN(CC1)c2nc(ccc2CNC(=O)Nc3ccc(CNS(=O)(=O)C)c(F)c3)C(F)(F)F\n",
+        "\n",
+        "\n",
+        "The formulation of the structure into linear form helps us immensely, and thus we are able to parse and reduce these complex molecules down to 2 dimensions using conventional statistical tools, namely UMAP. Ultimately we demonstrate how such an OPEN-SOURCE analysis can be sped-up and scaled-up massively with the [graphistry](graphistry.com) environment and toolkit\n"
+      ]
```

Comment:
Explicitly state the benefit as in the others:

* Speedup: From minutes to seconds (?) - Xmin to Ys on a small T4 GPU
* Visual insight: Add interactivity, similarity edges, and visual scale to a traditional static scatterplot to better investigate pairwise correlations and overall clusters